fe/lib/components/LocationSelectMap/queries.ts

```diff
@@ -0,0 +1,43 @@
+import { gql } from '@apollo/client';
+
+const LOCATION_ATTRIBUTES = gql`
+  fragment locationAttributes on Location {
+    id {
+      value
+    }
+    name
+    contextualName
+    countryCode
+  }
+`;
+
+const NESTED_LOCATION_ATTRIBUTES = gql`
+  fragment nestedLocationAttributes on Location {
+    ...locationAttributes
+    parent {
+      ...locationAttributes
+      parent {
+        ...locationAttributes
+        parent {
+          ...locationAttributes
+          parent {
+            ...locationAttributes
+            parent {
+              ...locationAttributes
+            }
+          }
+        }
+      }
+    }
+  }
+  ${LOCATION_ATTRIBUTES}
+`;
```

Comment:
Just curious, do we actually need this level of nesting? Is this because we feed the location in to another component that tries to display its full hierarchy?

Reply:
Yeah this level of nesting won't be used within the `LocationSelectMap` component, but is necessary for when we feed the location to other components. For example, in dev dashboard we display the full hierarchy in a `Breadcrumbs` component once a location is selected from the `LocationSuggest`